addons/website_slides/models/slide_slide.py

```diff
@@ -240,6 +240,12 @@ def _compute_questions_count(self):
         for slide in self:
             slide.questions_count = len(slide.question_ids)
 
+    def has_additional_resources(self):
```

Comment:
Woud make a private method, as otherwise it allows to leak a bit of information (knowing if an id exists and has resources)